src/test/java/reposense/authorship/FileAnalyzerTest.java

```diff
@@ -83,6 +79,8 @@ public void analyzeFile_blameTestFileIgnoreFakeAuthorCommitFullHash_success() {
 
     @Test
     public void analyzeFile_blameTestFileIgnoreAllCommit_success() {
+        config.setSinceDate(BLAME_TEST_SINCE_DATE);
+        config.setUntilDate(BLAME_TEST_UNTIL_DATE);
```

Comment:
As the current implementation checks if each line's commit date is within `sinceDate` and `untilDate`, config must be set with a `sinceDate` and `untilDate` before running `FileInfoAnalyzer.analyzeFile()`

Reply:
I see, but if it affects so many unit tests, should it be applied to all the unit test, under the GitTestTemplate.java using before or beforeClass?

Reply:
The `sinceDate` and `untilDate` of different tests are different tho. So I'm not sure if its suitable to place under `before` or `beforeClass`.

Reply:
>The sinceDate and untilDate of tests are different tho. So I'm not sure if its suitable to place under before or beforeClass.

Actually on second thought, why aren't we using full range?

Reply:
I’m not sure though I just followed the dates set from last time. But some test cases are to test when only since date is provided, some for when only until date is provided etc

Reply:
Only those that requires to be set, should be set though.
This change seems added that requirement to many other unit tests.